src/components/add-form/index.js

```diff
@@ -14,7 +14,6 @@ const AddForm = ({ formState, handleForm }) => (
         placeholder="Item"
         value={formState.productName}
         onChange={(event) => handleForm(event)}
-        style={{ marginTop: '8px' }}
```

Comment:
Nice to remove inline styling, it makes it harder to mantain :+1:

Reply:
I can't agree more! 😄 .